style.css

```diff
@@ -8,23 +8,58 @@ body {
   font-family: 'Inter', sans-serif;
 }
 
+.menu-list {
+  position: absolute;
+  left: 0;
+  transform: translateX(-100%);
+  width: 100%;
+  height: 100vh;
+  background: rgba(255, 255, 255, 1);
+  padding-left: 25px;
+  padding-top: 50px;
+  z-index: 1;
+  transition: transform 0.3s;
+  top: 0;
+}
```

Comment:
- [ ] KIndly make sure when the mobile menu is open even if we scroll we can only still see the mobile menu, on your portfolio we can see the other content when we scroll. Kindly make the mobile menu still covers the whole screen even if we scroll while it is open.
  _**You can achieve that by changing your position container to `fixed` and make sure to deal with the space around it.**_


   ### YOUR APP BEHAVIOUR 
   ![chrome-capture-2023-5-27](https://github.com/axcell-0/microverse-portfolio/assets/87197412/e1c35a32-3982-4648-a776-62d79dcb1543)